File: sentimental-mario-less/mario.py
# TODO

while (True):
    try:
        height = int(input("Enter Height: "))
        if (height >= 1 and height <= 8):
            break
    except ValueError:
        print("Invalid input.")

# create mario
for i in range(1, height + 1, 1):
    for j in range(height, 0, -1):
        if j <= i:
            print("#", end='')
        else:
            print(" ", end='')
    print("\n", end='')

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    do
    {
        Height = get_int("Enter height: ");
    }
    while (Height < 1 | Height > 8);

    for (int i = 1; i <= Height; i++)
    {
        for (int j = Height; j >= 1; j--)
        {
            if (j <= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}
